Content.Server/Atmos/Reactions/GasReactionPrototype.cs

```diff
@@ -60,6 +60,27 @@ public sealed partial class GasReactionPrototype : IPrototype
         /// </summary>
         [DataField("effects")] private List<IGasReactionEffect> _effects = new();
 
+        [DataField("enthalpy")]
+        public float Enthalpy;
+
+        /// <summary>
+        /// Integer gas IDs and integer ratios required in the reaction.
+        /// </summary>
+        [DataField("reactants")]
+        public Dictionary<Gas, int> Reactants = new();
+
+        /// <summary>
+        /// Integer gas IDs and integer ratios of reaction products.
+        /// </summary>
+        [DataField("products")]
+        public Dictionary<Gas, int> Products = new();
+
+        /// <summary>
+        /// Integer gas IDs and how much they modify the activation energy (J/mol).
+        /// </summary>
+        [DataField("catalysts")]
+        public Dictionary<Gas, int> Catalysts = new();
```

Comment:
These can be turned into arrays indexed by the Gas enum, as the rest of atmos code does, unless you are planning to add a substantial amount of gases.

Reply:
This is intended to make it easier to write reactions in YAML. It is planned for the generic gas reaction system to convert all the dictionaries into flat arrays on prototype reload, e.g. in the SIMD matrix version. It is not really a good reason to force YAML authors to use the array when that can be done automatically by the program especially since the dictionary would improve read/write/maintainability.

Reply:
Misunderstood the comment; yes, these should be turned into arrays indexed by gas enum on YAML reload.